### PR TITLE
MYMATH-1: Implement prime check

### PR DESCRIPTION
**Pull request by oszedinacubix**
Hey Colleagues, please review my changes. Thanks in advance.

### CODE REVIEW

File: is_prime.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def is_prime(n):
+    if n <= 1:
+        return False
+    for i in range(2, n):
```
Comment (oszedi) [inline is_prime.py:4]: no need to go till n, you could change to go till the square root

File: is_prime.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        if n % i == 0:
+            return False
+    return True
+
+for i in range(1, 20):
```
Comment (oszedi) [inline is_prime.py:9]: Add if __name__ == '__main__'

**Review by oszedi — CHANGES_REQUESTED** [review]
Nice code, but I have a few comments to consider. Furthermore you did a typo in your commit message, please fix that too.

File: is_prime.py (hunk 1 above)
Comment (oszedinacubix) [reply is_prime.py]: changed

File: is_prime.py (hunk 2 above)
Comment (oszedinacubix) [reply is_prime.py]: added

File: is_prime.py (hunk 1 above)
Comment (oszedi) [reply is_prime.py]: almost fine, but for square numbers it returns false, as it does not check the square root. Please add +1 to this.

Comment (oszedinacubix) [reply is_prime.py]: nice finding, thanks! added

**Review by oszedi — APPROVED** [review]
Nice change, you can now merge.

**Pull request merged**